indexer/packages/postgres/src/db/migrations/migration_files/20231101173444_update_fills_type_with_deleverage.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.raw(`
+    ALTER TABLE ONLY fills
+    DROP CONSTRAINT IF EXISTS fills_type_check;
+
+    ALTER TABLE ONLY fills
+    ADD CONSTRAINT fills_type_check
+    CHECK (type = ANY (ARRAY['MARKET'::text, 'LIMIT'::text, 'LIQUIDATED'::text, 'LIQUIDATION'::text, 'DELEVERAGED'::text, 'OFFSETTING'::text]));
```

Comment:
I believe this blocks reads/writes to the table until the constraint is checked.
Follow the example in this [migration](https://github.com/dydxprotocol/v4-chain/blob/main/indexer/packages/postgres/src/db/migrations/migration_files/20230726191736_add_orders_status_untriggered.ts#L5) to make the migration non-blocking.